includes/definitions/endrun.yaml

```diff
@@ -19,3 +19,8 @@ discovery:
           oid: .1.3.6.1.4.1.13827.12.1.16.0
           op: 'contains'
           value: 'Sonoma_D12'
+    -
+      snmpget:
+          oid: .1.3.6.1.4.1.13827.14.1.16.0
```

Comment:
Hi @hanserasmus 
You need to add 
```      sysObjectID: .1.3.6.1.4.1.8072.3.2.10```
above the snmpget line. Exactly like Tempus LX CDMA. 

And this should also be changed for Sonoma D12. 

The idea is that if (and only if) sysObjectID equals  .1.3.6.1.4.1.8072.3.2.10, then the snmpget will take place. It will ensure that this snmpget is not tried on all devices -> results in quicker discovery with less unnecessary snmpget for all other devices.

Reply:
Ok. Thank you @PipoCanaja  I will do so and resubmit. I see now that there is one failed test as well, will fix the indentation as well.